source/views/faqs/index.js

```diff
@@ -15,79 +15,62 @@ const styles = StyleSheet.create({
 	},
 })
 
-type Props = {}
+export function FaqView() {
+	let [text, setText] = useState('')
+	let [loading, setLoading] = useState(true)
+	let [refreshing, setRefreshing] = useState(false)
 
-type State = {
-	text: string,
-	loading: boolean,
-	refreshing: boolean,
-}
-
-export class FaqView extends React.PureComponent<Props, State> {
-	static navigationOptions = {
-		title: 'FAQs',
-	}
+	useEffect(() => {
+		fetchData().then(() => setLoading(false))
```

Comment:
This can use the wrong setLoading. You need to put [setLoading] as a dependency of the useEffect hook.

Reply:
Could you explain a little more?

Reply:
We chatted over the phone.

Reply:
I can only recall bits about promise cancellations. Did we chat about more than that?